test/EndToEndTests/Tests/Client/Build.Desktop/DeltaTests/DeltaTests.cs

```diff
@@ -48,20 +48,21 @@ public void RequestDeltaLink()
                 {
                     using (var messageReader = new ODataMessageReader(responseMessage, readerSettings, Model))
                     {
-                        var deltaReader = messageReader.CreateODataDeltaReader(customersSet, customerType);

```

Comment:
This change seems unrelated to this Pr, is it ?

Reply:
Yes it is. The build pipeline for 8.x is not running the E2E tests currently so a previous 8.x pull request got rid of `CreateODataDeltaReader` since it's deprecated and inadvertently forgot to update this test. I tried to build the E2E solution locally and it couldn't run so I fixed it in this pull request